sdk/sqlmanagement/Azure.ResourceManager.Sql/src/Generated/FailoverGroupData.cs

```diff
@@ -58,19 +58,7 @@ internal FailoverGroupData(ResourceIdentifier id, string name, ResourceType reso
         /// <summary> Read-write endpoint of the failover group instance. </summary>
         public FailoverGroupReadWriteEndpoint ReadWriteEndpoint { get; set; }
         /// <summary> Read-only endpoint of the failover group instance. </summary>
-        internal FailoverGroupReadOnlyEndpoint ReadOnlyEndpoint { get; set; }
```

Comment:
There was just one property within `FailoverGroupReadOnlyEndpoint`, so the generator flatten it out.
Now there are multiple properties within, it should not be flatten anymore.

The fix is to add the missing properties back to the [custom code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/sqlmanagement/Azure.ResourceManager.Sql/src/Custom/FailoverGroupData.cs), and add EB.Never to hide it from the user like the existing one in the custom code.